modules/gapi/test/s11n/gapi_s11n_tests.cpp

```diff
@@ -349,4 +349,16 @@ TEST_F(S11N_Basic, Test_Bind_RunArgs_MatScalar) {
     }
 }
 
+TEST_F(S11N_Basic, Test_Wchar_Size) {
+    cv::gimpl::s11n::ByteMemoryOutStream os;
+    const wchar_t a = '\xe4', b = '\xbd';
+    os << a << b;
+    EXPECT_EQ(os.data().size(), 8U);
```

Comment:
again, swap parameters for EQ checks.

Reply:
+1

Reply:
Fixed. Why again are we doing this?

Reply:
Parameters order for EQ checks is `(expected_value, actual_value)`.